src/libraries/System.Private.Xml.Linq/src/System/Xml/Linq/XContainer.cs

```diff
@@ -880,8 +880,8 @@ internal async Task ReadContentFromAsync(XmlReader r, LoadOptions o, Cancellatio
 
         private sealed class ContentReader
         {
-            private readonly NamespaceCache _eCache;
-            private readonly NamespaceCache _aCache;
+            private NamespaceCache _eCache;
+            private NamespaceCache _aCache;
```

Comment:
It looks like this occurred as part of moving these from being locals to being fields as part of https://github.com/dotnet/runtime/commit/d88ba5b82dd0d1164869fe6d5fbcd41c7a62f26c#diff-b2b608017bdbacf5e4484368d6e1fcb03a9ea3b68f4aece41da253f72826acf9R908. Would they be better again as locals? I'm not sure why they were moved to be fields, but they're increasing the size of the ContentReader object by being here.
cc: @krwq